src/components/entity/ha-state-label-badge.ts

```diff
@@ -166,6 +168,31 @@ export class HaStateLabelBadge extends LitElement {
     }
   }
 
+  private _computeUpperLabel(
+    domain: string,
+    state: HassEntity
+  ): string | undefined {
+    if (["device_tracker"].includes(domain) && state.attributes.battery) {
+      return state.attributes.battery + "%";
+    }
+
+    return undefined;
+  }
+
+  private _computeUpperLabelIcon(
```

Comment:
I played around a bit with an icon in the label and haven't decided if it looks good or not:
![image](https://user-images.githubusercontent.com/1287159/94983527-2ba4d780-0509-11eb-887c-c9a465083d06.png)
I think it takes away more than it adds though